src/REPLMode.jl

```diff
@@ -445,13 +445,16 @@ const helps = Dict(
     is modified.
     """, CMD_TEST => md"""
 
-        test [opts] pkg[=uuid] ...
+        test [-p|project]  [opts] pkg[=uuid] ...
+        test [-m|manifest] [opts] pkg[=uuid] ...
 
         opts: --coverage
 
     Run the tests for package `pkg`. This is done by running the file `test/runtests.jl`
-    in the package directory. The option `--coverage` can be used to run the tests with
-    coverage enabled.
+    in the package directory. If `project` is passed the versions in the current manifest
```

Comment:
Maybe "If called with `--project` the versions..." instead (same for `mainfest` to `--manifest` below)?